api/src/main/java/org/openmrs/module/fhir/api/util/FHIRMedicationRequestUtil.java

```diff
@@ -147,8 +147,11 @@ public static MedicationRequest generateMedicationRequest(org.openmrs.DrugOrder
         if (doseUnit != null) {
             dose.setUnit(doseUnit.getName().getName());
         }
-        dose.setValue(omrsDrugOrder.getDose());
-        dosage.setDose(dose);
+
```

Comment:
why this needs to be done?

Reply:
this is because some drug orders contains dose in null